pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -455,20 +454,15 @@ func TestSimpleDaemonSetLaunchesPods(t *testing.T) {
 // When ScheduleDaemonSetPods is enabled, DaemonSets without node selectors should
```

Comment:
Some other tests in this file need to pass both when ScheduleDaemonSetPods is enabled and disabled. Would you add that too?